l1-contracts/scripts/ci_deploy_contracts.sh

```diff
@@ -29,7 +29,7 @@ retry docker pull $CLI_IMAGE
 
 # remove 0x prefix from private key
 PRIVATE_KEY=${CONTRACT_PUBLISHER_PRIVATE_KEY#0x}
-docker run \
+retry docker run \
```

Comment:
This retry probably would not go perfectly well because we're hoping to `tee` all the input. You'd need either a custom loop or this whole thing to be a script

Reply:
I'd probably go custom loop - retry will wait max 30 seconds, this could take more?

Reply:
I moved it up a few steps as well, I think it should be fined. updated now!